packages/api/src/config/index.ts

```diff
@@ -1,5 +1,55 @@
 import { TypeOrmModuleOptions } from "@nestjs/typeorm";
 import * as featureFlags from "./featureFlags";
+import { BASE_TOKEN_L2_ADDRESS } from "../common/constants";
+type BaseToken = {
+  symbol: string;
+  decimals: number;
+  l1Address: string;
+  l2Address: string;
+  liquidity: number;
+  iconURL: string;
+  name: string;
+  usdPrice: number;
+};
+const defaultEthBaseToken: BaseToken = {
+  l2Address: BASE_TOKEN_L2_ADDRESS,
+  l1Address: "0x0000000000000000000000000000000000000001",
```

Comment:
Why 0x...0001?

```
import { utils } from "zksync-ethers";
console.log(utils.ETH_ADDRESS) //-> 0x...000
```

Reply:
It was outdated with some `kl-factory` changes. Changed now.